custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -68,6 +75,7 @@ def _base_properties(self, repeat_record):
         return {
             "regBy": username,
             "regby": username,
+            "RegBy": username,
```

Comment:
ya its different for some end point and i think same for two of the 4 end points. It would a dream come true if they make it a single representation of that. Have requested it.